File: C/1221.py
n = int(input())

for i in range(n):
  x = int(input())
  b = 0
  for a in range(2,x):
    if x % a == 0:
        print(x,'Not Prime') 
        b = 1
        break

  if b == 0:
    print(x,'Prime')

    

File: C/1221.c
#include <stdio.h>
/*
Questão 1221

Dupla
    Gabriel Costa de Moraes
    Matheus Seghatti de Moraes

numero primo

*/
int main(){
    int n;
    scanf("%d",&n);

    for (int i = 0; i < n; i++){
        int num;
        scanf("%d",&num);

        int primo = 1;
        for (int j = 2; j <= num / 2; j++){
                if (num % j == 0){
                    primo = 0;
                    break;
                }
        }
        printf("%s\n", primo ? "Prime" : "Not Prime");
        
    }
      
    return 0;
}
